src/app/config/parameters.yml

```diff
@@ -33,7 +33,11 @@ parameters:
 
     zikula.session.name: _zsid
     assets_version: ~ #e.g. 'v1'
-    
+    zikula_asset_manager.combine: true
+    zikula_asset_manager.lifetime: "1 day"
+    zikula_asset_manager.compress: false
+    zikula_asset_manager.minify: false
+
```

Comment:
because I don't think it should be *that* dynamic. This isn't a feature that needs to be enabled/disabled a bunch of times by the user. Additionally, using parameters separates the legacy methods from the new.